cmd/operator-sdk/bundle/validate.go

```diff
@@ -190,16 +208,34 @@ func (c bundleValidateCmd) run() (res internal.Result) {
 	// from the ValidateBundleContent to add the output(s) into the result
 	checkResults(results, &res)
 
-	return res
+	return res, nil
+}
+
+// newImageRegistryForTool returns an image registry based on what type of image tool is passed.
+// If toolStr is empty, a containerd registry is returned.
+func newImageRegistryForTool(logger *log.Entry, toolStr string) (reg registryimage.Registry, err error) {
+	switch tool := containertools.NewContainerTool(toolStr, containertools.NoneTool); tool {
+	case containertools.PodmanTool, containertools.DockerTool:
+		reg, err = execregistry.NewRegistry(tool, logger)
+	case containertools.NoneTool:
+		reg, err = containerdregistry.NewRegistry(
+			containerdregistry.WithLog(logger),
+			// In case reg.Destroy() fails in the caller, make it obvious where this cache came from.
+			containerdregistry.WithCacheDir(filepath.Join(os.TempDir(), "bundle-validate-cache")),
+		)
+	default:
+		err = fmt.Errorf("unrecognized image-builder option: %s", toolStr)
+	}
+	return reg, err
```

Comment:
- I passed an invalid value and it shows that still using the docker as tool besides I see the log `container-tool=invalid`. So, is it really working? If I have podman and docker installed how we can unsure and pass --image-builder=podman how can we ensure that the podman was used and not the docker for example?